test/components/plots/plotTests.ts

```diff
@@ -34,24 +34,24 @@ describe("Plots", () => {
       var r = new CountingPlot();
       r.addDataset("foo", dFoo);
 
-      assert.equal(1, r.renders, "initial render due to addDataset");
+      assert.strictEqual(1, r.renders, "initial render due to addDataset");
```

Comment:
We will need to do an audit soon over the testing infrastructure, as we have very little rules over it and well we should